js/stringProcessing/removeSentenceTerminators.js

```diff
@@ -9,5 +9,5 @@ var sentenceTerminators = /[.?!:;,]/g;
  * @returns {String} The sanitized text.
  */
 module.exports = function( text ) {
-	return text.replace( sentenceTerminators, "" );
+	return text.replace( sentenceTerminators, " " );
```

Comment:
Please revert this and solve the problem that has been solved with this function in `getWords` in another way.